balena.yml

```diff
@@ -1,5 +1,10 @@
 name: "Inkyshot"
 description: "Get a daily random inspirational quote delivered direct to your desk with Inkyshot. Build multiple Inkyshots and share the inspiration with your friends, family and loved ones ❤️"
+fleetcta: Get inspired
+post-provisioning: >-
+  # Usage instructions
```

Comment:
Can we make these has h2(`##`) instead?

Reply:
@iamsolankiamit we can, but should handle the case that the user puts in something else, as we won't be able to control it